src/frontend/Komponenter/Behandling/Brev/BrevFane.tsx

```diff
@@ -32,6 +34,9 @@ interface Props {
     behandling: Behandling;
 }
 
+const BREVMALER_SHAREPOINT_URL =
```

Comment:
Stusset litt på at det var URL ikke var i en form for config fil, men antar dette er greit siden det uansett blir samme url avhengig av preprod/prod.